### DISCUSSION TITLE
zh/guide/advanced/alias

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#302] [Announcements]
# zh/guide/advanced/alias

我们先看一张示意图 alias 我们看到两个不同的文件夹合并在一个文件夹里面了，名字一样的文件文件夹也都合并在一起了，独有的单独显示了 例1 riluo.jpg 是本地1 独有的，在上面显示了; 例2 两个文件夹都有video文件夹，但是里面的内容也会进行合并，子文件夹也同样适用于同名合并的展示规则; 两个文件夹分别有两个视频，但是有一个是同名的，最后...

https://alist.nn.ci/zh/guide/advanced/alias.html

**Comment by m0e16** [comment]
例2、根据下面的路径填写访问(302模式有优先权)
````
- 2，一个视频 2 4 有 那么每次访问的都是`4`
  - 如果 4 使用的本地代理，2 使用的`302`，优先调用 2
````
希望能说清楚2 4是根据填写路径排序的第2，第4行还是根据文件名的“本地2”和“本地4”

**Comment by se7enmuting** [comment]
实例中讲的“路径”应该是Alist“储存”中已经有的挂载路径，这点最好说明下。我一个纯本地存储的使用者，没挂网盘，设置“别名”设置了半天也合并不了文件夹，原因是我在“别名”的“路径”中直接使用了本机 liunx 系统的文件路径。。。

**Reply by LegendAdmin** [reply]
我说怎么设置半天是空的😕

**Comment by tact520** [comment]
我研究了半天才明白，我觉得这个指南漏了一个最重要的部分：alias别名在哪里设置

给同样没看明白的人：

它是这样的用的：管理-存储-添加，驱动选别名。。。

**Reply by yuanxiao1234** [reply]
顶上去，这个真的解决我问题了

**Comment by suixing8** [comment]
按照例3的操作，无法出现两个不同的文件夹来展示比如例子中
本地测试:/本地1/video  这个中使用本地测试，是无法使用的，直接加载不到目录，我使用本地:/本地1/video才可以，无法自由的自定义文件夹名称，这是有规定使用的文件夹名称吗？
比如这是我的alist目录结构
主页/文件夹1/文件夹A
主页/文件夹1/文件夹B
主页/文件夹2/文件夹C
主页/文件夹2/文件夹D
主页/文件夹3/文件夹E
主页/文件夹3/文件夹F
我想在别名中显示
主页/别名/文件夹1/...
主页/别名/文件夹3/...
但目前好像只能是像以下显示
主页/别名/文件夹A
主页/别名/文件夹B
主页/别名/文件夹E
主页/别名/文件夹F

**Comment by meteor614** [comment]
我设置好了，浏览也是正常的，文件夹也能合并显示，但是下载（包括发送到aria2也一样）和预览都不行。但复制编码后的url是可以下载的。两者路径不一样
